js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorerFilters.tsx

```diff
@@ -117,6 +117,11 @@ export function useAssetGraphExplorerFilters({
       </Box>
     ),
     getStringValue: (group) => group.groupName,
+    getTooltipText: (group) =>
+      group.groupName +
+      ' - ' +
+      buildRepoPathForHuman(group.repositoryName, group.repositoryLocationName),
```

Comment:
I'm not sure how we normally display asset group names within code locations, but the dash is somehow a little odd to me. Maybe `${groupName} in ${...}`?